### PR TITLE
Atlas update v2023_02

### PR DESCRIPTION
**Pull request by tomsercu**

### CODE REVIEW

File: scripts/atlas/README.md [hunk 1]
```diff
@@ -3,31 +3,37 @@
...
 - `num_conf` is the number of residues with plddt > 0.7
 - `len` is the total residues in the protein
+- `is_fragment` indicates whether the protein sequence is identified as a fragment in the MGnify90 sequence database.
+- `sequenceChecksum` is the CRC64 hash of the sequence. Can be used for cheaper lookups.
+- `esmfold_version` is the version of ESMFold, matching the model accessible as `esm.pretrained.esmfold_v{0,1}`
+- `atlas_version` is the Atlas version where this structure first appeared. Note: some of the predictions appearing for the first time in `v0` are also part of Atlas `v2023_02`.
 
-In parallel with `stats.parquet`, the sequences can be downloaded as fasta file from: <https://dl.fbaipublicfiles.com/esmatlas/v0/full/atlas.fasta>.
-The fasta file has `617051007` records matching the stats file, has file size 114GB, and has md5 hash `dc45f4383536c93f9d871facac7cca93`.
+Note you can remove records with missing structures with `df[~df.plddt.isna()]`.
```
Comment (w3ntinglu) [inline scripts/atlas/README.md:36]: do we wanna mention proteins longer than 1281 were not folded? (optional though as I do expect few other shorter proteins did not get folded due to edge cases)

File: scripts/atlas/README.md [hunk 2]
```diff
@@ -3,31 +3,37 @@
...
-The full database is available as PDB structures and is 15TB in size.
+The full database is available as PDB structures and is 15TB in size for `v0`.
 
-We also provide a metadata dataframe: <https://dl.fbaipublicfiles.com/esmatlas/v0/stats.parquet>.
-You can load the file with pandas: `df = pd.read_parquet('stats.parquet')`.
-The dataframe has length `617051007`, the file size is 6.0GB and has md5 hash `3948a44562b6bd4c184167465eec17de`.
-This dataframe has 4 columns:
+We also provide a metadata dataframe: <https://dl.fbaipublicfiles.com/esmatlas/v2023_02/metadata-rc1.parquet>.
+You can load the file with pandas: `df = pd.read_parquet('metadata.parquet')`.
+The dataframe has length 773,846,840 records, and the file size is around 25GB.
```
Comment (w3ntinglu) [inline scripts/atlas/README.md:24]: records # confirmed. I am not sure about the file size though, as I will be adding more metrics in. but not a big issue here

**Review by w3ntinglu — APPROVED** [review]
overall looks good!!

File: scripts/atlas/README.md (hunk 1 above)
Comment (tomsercu) [reply scripts/atlas/README.md:36]: Ah good point let me update now

File: scripts/atlas/README.md [hunk 3]
```diff
@@ -3,79 +3,79 @@
 The [ESM Metagenomic Atlas](https://esmatlas.com) is a repository of over database of more than 600 million metagenomic protein structures predicted by ESMFold.
```
Comment (w3ntinglu) [inline scripts/atlas/README.md:3]: do we want to update the `more than 600 million` stat? there are over 760M structures now

```
>>> len(df[~df.plddt.isna())
762985218
```
^ the actual count would be a bit higher as the backfill is still ongoing

File: scripts/atlas/README.md [hunk 4]
```diff
@@ -3,79 +3,79 @@
...
+For example, the foldseek database containing ptm from 0.60 to 0.70 and plddt from 0.80 to 0.90 is named `tm_.60_.70_plddt_.80_.90.DB`.
+The data are given as bundles of 500k or 1M structures each.
+
+* **Bulk Predicted structures**: see `{v0,v2023_02}/full/tarballs/`. The URLs for all shards of a single bundle will be in e.g. [v0/full/tarballs/tm_.60_.70_plddt_.80_.90.txt](v0/full/tarballs/tm_.60_.70_plddt_.80_.90.txt). The URLs for all tarballs across bins and their shards is available under `{v0,v2023_02}/full/tarballs.txt`.
+* **Foldseek DBs** `{v0,v2023_02}/full/foldseekdb/` and `{v0,v2023_02}/full/foldseekdb.txt`
+* **Bulk Embeddings (NEW - coming soon)** under `{v0,v2023_02}/full/esm2_embeddings/` and `{v0,v2023_02}/full/esm2_embeddings.txt`.
+  * Note: Individual LM embeddings can be fetched via the API endpoint `/fetchEmbedding/ESM2/:id` as described on <https://esmatlas.com/about#api>
+
+Note as of March 16, 2022: We expect to release bulk embeddings, and limited backfill of v2023_02 structures and foldseekdb, by adding URLs to the respective files.
+At that time we will update the metadata.sqlite file as well.
```
Comment (w3ntinglu) [inline scripts/atlas/README.md:59]: just realized the sqlite file was over 30G (54G), so I had to move it under `dl.fbaipublicfiles.com/large_objects/esmatlas/v2023_02/metadata-rc1.sqlite` instead. 

I also made a compressed sqlite file under the original directory, and uploaded to `dl.fbaipublicfiles.com/esmatlas/v2023_02/metadata-rc1.sqlite.gz`

Comment (tomsercu) [reply scripts/atlas/README.md:59]: Great, updated the links now.

**Pull request merged**